core/src/main/java/com/graphhopper/routing/lm/LandmarkStorage.java

```diff
@@ -765,20 +782,26 @@ public boolean apply(int nodeId, SPTEntry value) {
             return failed.get();
         }
 
-        public void initLandmarkWeights(final int lmIdx, final long rowSize, final int offset) {
+        public void initLandmarkWeights(final int lmIdx, int lmNodeId, final long rowSize, final int offset) {
             IntObjectMap<SPTEntry> map = from ? bestWeightMapFrom : bestWeightMapTo;
             final AtomicInteger maxedout = new AtomicInteger(0);
+            final Map.Entry<Double, Double> finalMaxWeight = new MapEntry<>(0d, 0d);
 
             map.forEach(new IntObjectProcedure<SPTEntry>() {
                 @Override
                 public void apply(int nodeId, SPTEntry b) {
-                    if (!lms.setWeight(nodeId * rowSize + lmIdx * 4 + offset, b.weight))
+                    if (!lms.setWeight(nodeId * rowSize + lmIdx * 4 + offset, b.weight)) {
                         maxedout.incrementAndGet();
+                        finalMaxWeight.setValue(Math.max(b.weight, finalMaxWeight.getValue()));
+                    }
                 }
             });
 
-            if ((double) maxedout.get() / map.size() > 0.1)
-                LOGGER.warn("landmark " + lmIdx + " -> too many weights were maxed out (" + maxedout.get() + "/" + map.size() + "). Use a bigger factor " + lms.factor);
+            if ((double) maxedout.get() / map.size() > 0.1) {
+                LOGGER.warn("landmark " + lmIdx + " (" + nodeAccess.getLatitude(lmNodeId) + "," + nodeAccess.getLongitude(lmNodeId) + "): " +
+                        "too many weights were maxed out (" + maxedout.get() + "/" + map.size() + "). Use a bigger factor than " + lms.factor
+                        + ". For example use the following in the config.properties: weighting=" + weighting.getName() + "|maximum=" + finalMaxWeight.getValue() * 1.2 / PRECISION);
```

Comment:
Mhm, somehow the proposal does not match here :/. Why do we divide by PRECISION?

When I set `prepare.lm.weightings=fastest|maximum=100` then I get the following log `landmark 15 (45.5060110354277,-73.5529020105206): too many weights were maxed out (294/440). Use a bigger factor than 0.00152587890625. For example use the following in the config.properties: weighting=fastest|maximum=0.004969158898255066`.